test/uffizzi/cli/preview_test.rb

```diff
@@ -252,6 +252,7 @@ def test_create_preview_with_file_success
 
     # rubocop:disable Layout/LineLength
     expected_data = {
+      creation_source: nil,
```

Comment:
Let's add the source to the options and check that it's present in the expected data